src/include/cache.h

```diff
@@ -44,6 +44,9 @@ struct __wt_evict_worker {
 	uint32_t flags;
 };
 
+#define	WT_EVICT_CHECK_SPLITS	1	/* Boolean to __wt_page_can_evict  */
+#define	WT_EVICT_EXCLUSIVE	1	/* Boolean to __wt_evict */
+
```

Comment:
AFAIK, this is the only place we have macros for booleans.  I didn't mind the previous flags arrangement, but this change makes it half flags, half booleans-with-names, which seems strange to me.

Can we just pass 1 if we're going with booleans?  If those calls are unclear, maybe we should go back to flags?